bedrock/mozorg/templates/mozorg/home/includes/m24/hero.html

```diff
@@ -7,8 +7,12 @@
 <div class="m24-t-dark m24-t-alt" data-m24-hero-animation="{{ switch('m24-hero-animation') | lower }}">
   <section class="m24-c-flag" data-animation-running="false">
     <button class="m24-c-flag-button js-animation-button" aria-live="assertive">
-      <span class="m24-c-flag-button-pause"><img src="{{ static('img/m24/icon-pause.svg') }}" alt=""> <span class="m24-c-flag-button-text">{{ ftl('ui-pause-animation') }}</span></span>
-      <span class="m24-c-flag-button-play"><img src="{{ static('img/m24/icon-play.svg') }}" alt=""> <span class="m24-c-flag-button-text">{{ ftl('ui-play-animation') }}</span></span>
+      <span class="m24-c-flag-button-pause">
+        <svg width="14" height="14" viewBox="0 0 14 14" fill="none" aria-hidden="true" focusable="false" xmlns="http://www.w3.org/2000/svg"><path d="M4 1v12M10 1v12" stroke="currentColor" stroke-width="2"/></svg>
```

Comment:
Using currentColor here to reflect the text color changes from different states. Requires inline SVG. Might be overusing that method of including SVGs though.

Other options:

1. external asset + [image-mask](https://caniuse.com/?search=image-mask)
2. separately colored external assets, swapping with state change